pages/levels.php

```diff
@@ -79,8 +79,8 @@
 </table>
 <p class="<?php echo pmpro_get_element_class( 'pmpro_actions_nav' ); ?>">
 	<?php if( $has_any_level ) { ?>
-		<a href="<?php echo pmpro_url("account")?>" id="pmpro_levels-return-account"><?php _e('&larr; Return to Your Account', 'paid-memberships-pro' );?></a>
+		<a href="<?php echo esc_url( pmpro_url("account" ) ) ?>" id="pmpro_levels-return-account"><?php esc_html_e('&larr; Return to Your Account', 'paid-memberships-pro' );?></a>
```

Comment:
This won't work since it has entities built in

```suggestion
		<a href="<?php echo esc_url( pmpro_url("account" ) ) ?>" id="pmpro_levels-return-account">&larr; <?php esc_html_e('Return to Your Account', 'paid-memberships-pro' );?></a>
```